compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -120,22 +124,24 @@ class TextInputTests : OnCanvasTests {
             keyEvent("1")
         )
 
-        assertEquals("step1", inputInteractor.waitForInput())
+        awaitIdle()
```

Comment:
Consider adding `awaitIdle` to the end of `sendToHtmlInput` (or add a new method, `sendToHtmlInputAndAwaitIdle`. Typically you want to wait for recomposition after sending inputs.